cryptoutil/cryptoutil.go

```diff
@@ -0,0 +1,54 @@
+// Package cryptoutil provides a set of functions for 2-way encryption and hashing.
+package cryptoutil // import "github.com/teamwork/utils/cryptoutil"
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"encoding/base64"
+	"io"
+)
+
+// Encrypt binary data to a base64 string with AES using the key provided.
+func Encrypt(keyString string, data []byte) (string, error) {
+	key := []byte(keyString)
+
+	block, err := aes.NewCipher(key)
+	if err != nil {
+		return "", err
+	}
+
+	ciphertext := make([]byte, aes.BlockSize+len(data))
+	iv := ciphertext[:aes.BlockSize]
+	if _, err := io.ReadFull(rand.Reader, iv); err != nil {
+		return "", err
+	}
+
+	stream := cipher.NewCFBEncrypter(block, iv)
+	stream.XORKeyStream(ciphertext[aes.BlockSize:], data)
+
+	return base64.URLEncoding.EncodeToString(ciphertext), nil
+}
+
+// Decrypt a base64 string to binary data with AES using the key provided.
+func Decrypt(keyString string, base64Data string) ([]byte, error) {
+	key := []byte(keyString)
+	ciphertext, _ := base64.URLEncoding.DecodeString(base64Data)
+
+	block, err := aes.NewCipher(key)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(ciphertext) < aes.BlockSize {
+		return nil, err
```

Comment:
This will return `nil, nil`. Probably want to return `errors.New("...")`

Reply:
Good catch 👍